src/views/OnBoardingSubProduct/components/SubProductStepSelectUserParty.tsx

```diff
@@ -18,21 +18,24 @@ export function SubProductStepSelectUserParty({ forward, parties }: Props) {
 
   const theme = useTheme();
 
+  const [origin, setOrigin] = useState<string>('');
   const [selected, setSelected, setSelectedHistory] = useHistoryState<SelfcareParty | null>(
     'SubProductStepSelectUserParty',
     null
   );
   const onForwardAction = () => {
     setSelectedHistory(selected);
-    forward(selected as Party);
+    forward(selected as SelfcareParty);
```

Comment:
Party is the right model to use here
```suggestion
    forward(selected as Party);
```